server/controllers/game.ts

```diff
@@ -74,6 +74,23 @@ gameController.get({ path: '/play', userType: UserType.TEACHER }, async (req: Re
   res.sendJSON(game);
 });
 
+//--- Get number of games available ---
+gameController.get({ path: '/isGameCreateBeforePlaying', userType: UserType.TEACHER }, async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Adding this works, but we should avoid having too many API routes. A good pattern is to have only 5 routes for a ressource:
- `GET /` Get all
- `GET /<id>` Get one
- `POST /` Create one
- `PUT /<id>` Update one
- `DELETE /<id>` Delete one

We already added '/play' and '/ableToPlay' that returns specific games. I'm ok with these ones.

But for the new request you are adding, you are returning the count of the user's games. This can be done in the get all games request with a url parameters: GET `/?userId=<user_id>` or GET `/?userId=self`. 

So instead of adding a new request, you can update the first request `//--- Get all games ---` line 15 to accept url parameters to filter on the games to return. And then the frontend would get the count with the size of the list.

Reply:
If you want you can do that in another PR